ROADMAP.md

```diff
@@ -6,36 +6,21 @@ targeting these areas are very welcome, but please check first with a
 core team member that nobody else is working on the same thing.
 
 **Note:** This doesn’t include everything that the core team will work
-on, and everything is subject to change. Near-term plans are likely
-more accurate.
-
-## September-December 2017
-
-- Fix remaining highest-priority TypedDict issues and make TypedDict
-  an officially supported mypy feature.
-
-- Add support for protocols and structural subtyping (PEP 544).
-
-- Continue making error messages more useful and informative.
-  ([issue](https://github.com/python/mypy/labels/topic-usability))
-
-- Switch completely to pytest and remove the custom testing framework.
-  ([issue](https://github.com/python/mypy/issues/1673))
+on, and everything is subject to change.
 
 - Make it possible to run mypy as a daemon to avoid reprocessing the
   entire program on each run. This will improve performance
   significantly. Even when using the incremental mode, processing a
   large number of files is not cheap.
 
 - Provide much faster, reliable interactive feedback through
-  fine-grained incremental type checking, built on top the daemon
+  fine-grained incremental type checking, built on top of the daemon
```

Comment:
Capital 'F'.

Reply:
We use a lower-case 'F' elsewhere (in "fine-grained incremental ...", so retaining it here for consistency.